x-pack/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/edit_job_flyout.js

```diff
@@ -132,13 +136,13 @@ export class EditJobFlyout extends Component {
       jobGroupsValidationError = validateGroupNames(jobDetails.jobGroups).message;
     }
 
-    const valid = (jobModelMemoryLimitValidationError === '' && jobGroupsValidationError === '');
+    const validJobDetails = (jobModelMemoryLimitValidationError === '' && jobGroupsValidationError === '');
 
     this.setState({
       ...jobDetails,
       jobModelMemoryLimitValidationError,
       jobGroupsValidationError,
-      valid,
+      validJobDetails,
```

Comment:
this variable name makes it sound like a filtered list of valid job details.
perhaps something like `jobDetailsValid` or `isJobDetailsValid` would be clearer